metaflow/tutorials/07-worldview/README.md

```diff
@@ -6,16 +6,5 @@ monitor all of your Metaflow flows.**
 #### Showcasing:
 - The metaflow client API.
 
-#### Before playing this episode:
-1. ```python -m pip install notebook```
-2. This tutorial requires access to compute and storage resources on AWS, which
-   can be configured by 
-   1. Following the instructions at 
-      https://docs.metaflow.org/metaflow-on-aws/deploy-to-aws or
-   2. Requesting a sandbox at 
-      https://docs.metaflow.org/metaflow-on-aws/metaflow-sandbox
-
 #### To play this episode:
-1. ```cd metaflow-tutorials```
-2. ```jupyter-notebook 07-worldview/worldview.ipynb```
-3. Open 'worldview.ipynb' in your remote Sagemaker notebook
+1. Open ```07-worldview/worldview.ipynb```
```

Comment:
install instructions for notebook have been removed, so is the assumption going forward that notebooks are installed due to the user running through earlier tutorials?

Reply:
Yes, but for consistency I think it makes sense to add back in, just removing part about Sagemaker. Most likely if people follow in Outerbounds sandbox, this will just be a VSCode notebook in the cloud workstation.